spec/ics-23-standard-accumulator/README.md

```diff
@@ -0,0 +1,228 @@
+---
+ics: 23
+title: Standard Accumulator
+stage: draft
+category: ibc-core
+author: Christopher Goes <cwgoes@tendermint.com>
+created: 2019-04-16
+modified: 2019-04-25
+---
+
+# Synopsis
+
+An *accumulator*, or *cryptographic accumulator*, is a construction that produces a succinct, binding commitment to a set or indexed vector of elements and short membership and/or non-membership proofs for any element in the set or vector.
+This specification enumerates the functions and properties required of accumulator constructions used in the IBC protocol.
+
+# Specification
+
+## Motivation
+
+In order to provide a guarantee of a particular state transition having occurred on one chain which can be verified on another chain, IBC requires an efficient cryptographic construction to prove inclusion or non-inclusion of particular values in state.
+
+## Definitions
+
+The *manager* of an accumulator is the actor with the ability and responsibility to add or remove items from the accumulator. Generally this will be the state machine of a blockchain.
+
+The *prover* is the actor responsible for generating accumulator proofs of inclusion or non-inclusion of particular elements. Generally this will be a relayer (see [ICS 18](../ics-18-relayer-algorithms)).
+
+The *verifier* is the actor who checks accumulator proofs in order to verify that the manager of the accumulator did or did not add a particular element. Generally this will be an IBC handler running on another chain.
+
+Accumulators are instantiated with a particular *element* type, which is assumed to be arbitrary serializable data.
+
+## Desired Properties
+
+This document only defines desired properties, not a concrete implementation — see "Properties" below.
+
+## Technical Specification
+
+### Datatypes
+
+An accumulator construction MUST specify the following datatypes, which are otherwise opaque (need not be introspected) but must be serializable:
+
+#### State
+
+An `AccumulatorState` is the full state of the accumulator, which will be stored by the manager.
+
+```golang
+type AccumulatorState struct
+```
+
+#### Root
+
+An `AccumulatorRoot` commits to a particular accumulator state and should be succinct.
```

Comment:
Same question, I'm wondering if "commitment/commit" is the most explicit word we can use for the root of the accumulator

Reply:
I think it's standard - but open to other suggestions.

Reply:
Ok, wasn't sure it was standard. Would be good to define it somewhere still imo

Reply:
For now, we'll get rid of "accumulator" and just call it "commitment".